src/chapters/strings/string-conditionals.rst

```diff
@@ -88,17 +87,23 @@ We can use the comparison operators ``==, !=, <, >, <=, >=`` on strings.
      character to the same case (e.g.
      ``your_pet[0].lower() == other_pet[0].lower()``).
 
-#. Line 7 evaluates if ``'dog'`` comes earlier in the alphabet than ``'cat'``.
-   A string that comes earlier is considered *less than* a string that comes
+#. Line 7 evaluates if ``'dog'`` is less than ``'cat'``. This comparison is a bit more complicated.
+   What is really being compared is each character's :ref:`Unicode <unicode>` value. All characters, even letters,
+   have Unicode values. The Unicode value of a letter corresponds with its alphabetical order. 
+   So a string that comes earlier in the alphabet is considered *less than* a string that comes
```

Comment:
I'm still struggling with this paragraph.

I see the new reference to the Unicode section, but the text there talks about special symbols and non-Latin characters.  Can we insert a simple mention here about Unicode relating to numerical values?

We don't have to go into anything like 0x0042 <---> 66 <---> "B", but helping the kids see that letters relate to numbers will clarify how the comparisons work.

Reply:
I just kept getting hung up on "Unicode values" not clicking in the students' brains as being comparable. Calling then numbers and giving the link to the interactive table helps.